docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: "2.1"
```

Comment:
What is this docker-compose used for vs the test one?

Reply:
Bascially it's there to fulfil this balean ci script: https://github.com/product-os/scripts/blob/master/docker/build.sh

Thinking about it, I need to add a specific script: npm + docker-compose testing.
As the output of the balena CI is a npm package but the testing needs docker-compose, this case is not yet covered in the scripts.

I'm going to work this out.

Reply:
Both files are needed to fulfil CI requirements. Thus, am proceeding with both, and we need in the future to either add a CI script to handle the case or utilise transformers for testing this repo with a docker-composition

Reply:
Actually I realized it'd make sense to put an explanation as a comment in the file itself for anyone looking at the file in future to know that it's just a workaround